.github/workflows/release-build.yml

```diff
@@ -52,6 +52,9 @@ jobs:
     - if: runner.os != 'Windows'
       run: ls -shal packages/
 
+    - name: Ensure release binary is runnable
```

Comment:
Can't do that on most cross compiles

Reply:
Oh I forgot to add the`use-cross` check

Reply:
Thanks, I've fixed this by adding a `if : "! matrix.c"` check.

Reply:
Aha turns out that we cannot run aarch64 macos/windows, also we did not use cross for `aarch64-unknown-linux-gnu`.

Reply:
So I have to introduce a new attribute in matrix `r` to specify whether to run the binary after built.